internal/outputgenerators/templates/cluster_settings.txt

```diff
@@ -39,7 +39,7 @@ Group.nrofApplications = 0
 Group{{add $index 1}}.groupID = {{$group.Id}}
 Group{{add $index 1}}.nrofHosts = {{$group.NrofHosts}}
 Group{{add $index 1}}.nrofInterfaces = {{$group.NrofInterfaces}}
-{{- if eq .DefaultPosition true}}
+{{- if eq $group.PredefinePosition true}}
```

Comment:
Typo: It should be Predefine***d***Position. PLEASE test your code before you request a review